.github/workflows/pr-benchmarks.yml

```diff
@@ -0,0 +1,35 @@
+name: "PR Benchmark workflow"
+on:
+  # Trigger benchmark test on this PR's commit against master.
+  pull_request:
+jobs:
+  benchmark:
+    name: Performance regression check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Install go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.17.9'
+      - name: Build go-algorand
+        run: scripts/travis/build.sh
+      - name: Run benchmark
+        run: go test ./data/transactions/logic -bench 'BenchmarkUintMath' | tee benchmark_output.txt
+      - name: Download previous benchmark data
+        uses: actions/cache@v1
+        with:
+          path: ./cache
+          key: ${{ runner.os }}-benchmark
+      - name: Store benchmark result
+        uses: benchmark-action/github-action-benchmark@v1
+        with:
+          name: Go Benchmark
+          tool: 'go'
+          output-file-path: benchmark_output.txt
+          # Where the previous data file is stored
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # Show alert with commit comment on detecting possible performance regression
+          alert-threshold: '200%'
```

Comment:
@algochoi Based on your experience so far, what's the lowest (i.e. tightest) threshold you'd feel comfortable configuring that minimizes false positives?

I'm sanity checking how deliberately the PR applies a 200% threshold.  From my spot check of prior results, it _seems_ a value as low as 160% is palatable.

Reply:
200% is probably a good threshold. 

There was a run where we reached 160% on the `exp` benchmark test due to some machine variance: https://github.com/algochoi/go-algorand/pull/10#commitcomment-73875077

Reply:
@algochoi Alright, I'm good to start as shown - resolving.